plugins/woocommerce/includes/admin/settings/class-wc-settings-emails.php

```diff
@@ -191,6 +191,18 @@ protected function get_settings_for_default_section() {
 					'desc_tip' => true,
 				),
 
+                array(
```

Comment:
Getting a PHPCS error here due to using spaces instead of tabs: https://github.com/woocommerce/woocommerce/actions/runs/9980468073/job/27582309298?pr=49648#step:4:30